### PR TITLE
Pin gha ubuntu version to 20.04

### PR DESCRIPTION
**Pull request by Baccata**
Attempts to address libc incompatibility errors experienced in CIs

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -124,7 +124,7 @@ jobs:
 
   release:
     timeout-minutes: 15
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```
Comment (sideeffffect) [inline .github/workflows/build.yml:127]: Wouldn't it be enough to pin it only here for the `release` job?

Comment (Baccata) [reply .github/workflows/build.yml:127]: I think it's better to keep a consistent version. Moreover, the release phase benefits from saved artifacts from earlier phases, see https://github.com/oyvindberg/bleep/pull/236/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R118-R123

Comment (oyvindberg) [reply .github/workflows/build.yml:127]: Exactly, let's keep it consistent! :)

**Comment by oyvindberg** [comment]
whaat, the github action checks didn't run, and it seems this wasn't as easy as this. for one thing the old binary doesn't work, and scalafmt doesn't work. pff

**Comment by oyvindberg** [comment]
merged on mobile so I didn't see that

**Comment by sideeffffect** [comment]
Should we start enforcing the protection of the `master` branch?

**Comment by oyvindberg** [comment]
Absolutely, doing it now. I hadn't really thought about what it meant that it was disabled :)

**Pull request merged**